src/assets/CDN_IMAGES.ts

```diff
@@ -0,0 +1,21 @@
+const CDN_BASE = import.meta.env.VITE_CLOUDINARY_BASE_URL;
+
+export const CDN_IMAGES = {
```

Comment:
attatrch a lighthouse report showing if we have made improvements in the image delivery department